api/requirements.txt

```diff
@@ -0,0 +1,104 @@
+absl-py==1.4.0 ; python_version >= "3.8" and python_version < "3.11"
+anyio==3.6.2 ; python_version >= "3.8" and python_version < "3.11"
+astunparse==1.6.3 ; python_version >= "3.8" and python_version < "3.11"
+attrs==22.2.0 ; python_version >= "3.8" and python_version < "3.11"
+blis==0.7.9 ; python_version >= "3.8" and python_version < "3.11"
+cachetools==5.3.0 ; python_version >= "3.8" and python_version < "3.11"
+catalogue==2.0.8 ; python_version >= "3.8" and python_version < "3.11"
+certifi==2022.12.7 ; python_version >= "3.8" and python_version < "3.11"
+chardet==4.0.0 ; python_version >= "3.8" and python_version < "3.11"
+click==8.1.3 ; python_version >= "3.8" and python_version < "3.11"
+colorama==0.4.6 ; python_version >= "3.8" and python_version < "3.11" and sys_platform == "win32" or python_version >= "3.8" and python_version < "3.11" and platform_system == "Windows"
```

Comment:
What's this line about? It looks like it won't run on Linux, which is what the server is running. Or am I not understanding?